test/utils/image/manifest.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"fmt"
+	"runtime"
+)
+
+const (
+	e2eImagesRegistry  = "gcr.io/kubernetes-e2e-test-images"
+	BusyBox            = "busybox"
```

Comment:
I'm fine with the any option, what do you recommend?
Option 1 - Refer busybox from docker hub
Option 2 - Pull busybox images for all the architecture and keep it in gcr.

My vote is for option 1 to avoid any maintenance required for pushing in gcr for all the architecture.!